SubmittedAssignments/Sanjeevanee_balure_03_/index.html

```
@@ -0,0 +1,809 @@
+      </div>
+      <div class="stat">
+        <div class="stat-number">3</div>
+        <p>Culinary Awards</p>
+      </div>
+      <div class="stat">
+        <div class="stat-number">15000+</div>
+        <p>Happy Customers</p>
+      </div>
+    </div>
+  </div>
+   </portfolio-section>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </portfolio-section> ]

(tag-pair)